.gitlab-ci.yml

```diff
@@ -233,6 +233,17 @@ windows32:
   variables:
     ARCH: "32"
 
+nixos:
```

Comment:
name this `distro:nixos` please.

Reply:
So maybe `pkg:nixos` ?

Based on what you say we would for example also add a test to build an opam package, so we would have `pkg:opam` and `pkg:nixos`. How does that sound?